Cargo.toml

```diff
@@ -24,15 +24,19 @@ rustc-args = ["--cfg", "docsrs"]
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
-default = ["file-drop", "objc-exception", "protocol"]
+default = ["file-drop", "objc-exception", "protocol", "native"]
 objc-exception = ["objc/exception"]
 file-drop = []
 protocol = []
 devtools = []
 transparent = []
 fullscreen = []
-linux-body = ["webkit2gtk/v2_40"]
+linux-body = ["webkit2gtk/v2_40", "native"]
 mac-proxy = []
+native = ["javascriptcore-rs", "webkit2gtk", "webkit2gtk-sys", "dep:gtk", "soup3", "x11-dl", "gdkx11"]
```

Comment:
what does `native` mean here? specially since it's enabling gtk

Reply:
I was thinking about macOS and Windows support in the future. They may also have servo and cef ports.
With `native` feature, it indicates existing frameworks from platforms.
And `gtk` is cfg alias only will be used in private to indicate it's native and on Linux.
This could be easier for us to configure internally.
And for public they just need to know, there's a native feature.

Reply:
Maybe `os-webview` is a better name?